notebooks/Media-Pretrained/02_Training_Layer_Recap.ipynb

```diff
@@ -111,9 +87,1051 @@
+    "    print(\"File {} uploaded to bucket {}\".format(users_filename, bucket_name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create the Dataset Group <a class=\"anchor\" id=\"group_dataset\"></a>\n",
+    "[Back to top](#top)\n",
+    "\n",
+    "The highest level of isolation and abstraction with Amazon Personalize is a *dataset group*. Information stored within one of these dataset groups has no impact on any other dataset group or models created from one - they are completely isolated. This allows you to run many experiments and is part of how we keep your models private and fully trained only on your data. \n",
+    "\n",
+    "Before importing the data prepared earlier, there needs to be a dataset group and a dataset added to it that handles the interactions.\n",
+    "\n",
+    "Dataset groups can house the following types of information:\n",
+    "\n",
+    "* User-item-interactions\n",
+    "* Event streams (real-time interactions)\n",
+    "* User metadata\n",
+    "* Item metadata\n",
+    "\n",
+    "We need to create the dataset group that will contain our three datasets."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create the Dataset Group\n",
+    "\n",
+    "The following cell will create a new dataset group with the name `personalize-poc-movielens`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_get_dataset_group_arn('workshop-personalize-poc-movielens')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workshop_dataset_group_name = 'workshop-personalize-poc-movielens'\n",
+    "\n",
+    "try: \n",
+    "    create_dataset_group_response = personalize.create_dataset_group(\n",
+    "        name = workshop_dataset_group_name,\n",
+    "        domain='VIDEO_ON_DEMAND'\n",
+    "    )\n",
+    "\n",
+    "    workshop_dataset_group_arn = create_dataset_group_response['datasetGroupArn']\n",
+    "    print(json.dumps(create_dataset_group_response, indent=2))\n",
+    "    print ('\\nCreating the Dataset Group with dataset_group_arn = {}'.format(workshop_dataset_group_arn))\n",
+    "\n",
+    "except personalize.exceptions.ResourceAlreadyExistsException as e:  \n",
+    "    workshop_dataset_group_arn = 'arn:aws:personalize:'+region+':'+account_id+':dataset-group/'+workshop_dataset_group_name \n",
+    "    print ('\\nThe the Dataset Group eith dataset_group_arn = {} already exists'.format(workshop_dataset_group_arn))\n",
+    "    print ('\\nWe will be using the existing Dataset Group dataset_group_arn = {}'.format(workshop_dataset_group_arn))\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Wait for Dataset Group to Have ACTIVE Status \n",
```

Comment:
Use consistent capitalization in headings.  Either all cap except connector words or sentence capitalization.  I prefer the latter.

_Wait for Dataset Group to have ACTIVE status

Reply:
Fixing!